generators/fastlane-setup/templates/deploy.sh

```diff
@@ -0,0 +1,106 @@
+#! /bin/bash
+set -e
+
+DEV=0
```

Comment:
I'm OK to merge this file but having such a deploy script has proved useless because deployments are made in a CD service (Bitrise)

Reply:
We use this when we want to quickly iterate over some deployment problems or with codepush. We also usually don't deploy the dev app with the ci but from local instead. Also on some projects like the one in Greece, the client doesn't allow using a CI since they want the all the code to stay in-house.

The use case is not huge but I appreciate the convenience and the checks prevent to deploy crap from local :)